src/macaron/slsa_analyzer/pypi_heuristics/metadata/unchanged_release.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+"""Heuristic analyzer to check unchanged content in multiple releases."""
+
+from collections import Counter
+
+from macaron.slsa_analyzer.package_registry.pypi_registry import PyPIApiClient
+from macaron.slsa_analyzer.pypi_heuristics.analysis_result import RESULT
+from macaron.slsa_analyzer.pypi_heuristics.base_analyzer import BaseAnalyzer
+from macaron.slsa_analyzer.pypi_heuristics.heuristics import HEURISTIC
+
+
+class UnchangedReleaseAnalyzer(BaseAnalyzer):
+    """Analyzer checks heuristic."""
+
+    def __init__(self, api_client: PyPIApiClient) -> None:
+        super().__init__(
+            name="unchanged_release_analyzer",
+            heuristic=HEURISTIC.UNCHANGED_RELEASE,
+            depends_on=[(HEURISTIC.HIGH_RELEASE_FREQUENCY, RESULT.FAIL)],  # Analyzing when this heuristic fail
+        )
+        self.hash: str = "sha256"
```

Comment:
nit:
```suggestion
        self.hash_algo: str = "sha256"
```

Reply:
Fixed this in 1d801c45b3f13c3cdb3e603935c314b1687cebc3